ERDDAP_timeseries_explorer.ipynb

```diff
@@ -522,6 +522,77 @@
     "standard_names = df[\"Category\"].values"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This cell identifies the valid standard names for the specified server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Checking the variables available for this server. This might take up to a couple of minutes...\n",
+      "\n",
+      "Halfway there...\n",
+      "\n",
+      "Almost done...\n",
+      "\n",
+      "Done!\n"
+     ]
+    }
+   ],
+   "source": [
+    "valid_standard_names = []\n",
+    "\n",
+    "count = 0\n",
+    "\n",
+    "print(\n",
+    "    \"Checking the variables available for this server. This might take up to a couple of minutes...\\n\"\n",
+    ")\n",
+    "\n",
+    "for standard_name in standard_names:\n",
+    "\n",
+    "    count += 1\n",
+    "\n",
+    "    if count == np.floor(len(standard_names) / 2):\n",
+    "        print(\"Halfway there...\\n\")\n",
+    "    elif count == np.floor((len(standard_names) / 4) * 3):\n",
+    "        print(\"Almost done...\\n\")\n",
+    "    elif count == (len(standard_names)):\n",
+    "        print(\"Done!\")\n",
+    "\n",
+    "    try:\n",
+    "\n",
+    "        features, datasets = stdname2geojson(\n",
+    "            e,\n",
+    "            standard_name,\n",
+    "            server.get(\"cdm_data_type\"),\n",
+    "            server.get(\"min_time\"),\n",
+    "            server.get(\"max_time\"),\n",
+    "        )\n",
+    "\n",
+    "        var = e.get_var_by_attr(\n",
+    "            dataset_id=datasets[0],\n",
+    "            standard_name=lambda v: str(v).lower() == standard_name.lower(),\n",
+    "        )\n",
+    "\n",
+    "        if var != []:\n",
+    "            valid_standard_names.append(standard_name)\n",
+    "\n",
+    "        del var, features, datasets\n",
+    "\n",
+    "    except NameError:\n",
+    "        continue\n",
+    "del count, standard_names, standard_name"
```

Comment:
Here the code is checking if the first dataset_id (which would be equivalent to station) of the server has data for the specified standard_name. There is a chance the first dataset_id does not have data for that standard_name, but other dataset_id (of the same server) might. I'll try to find a better way of doing this in a future PR.